pkg/auth0/api/token.go

```diff
@@ -3,11 +3,16 @@ package api
 import "time"
 
 type Token struct {
-	Type                string    `json:"token_type"`
-	Scope               string    `json:"scope"`
-	Identity            string    `json:"id_token"`
-	Access              string    `json:"access_token"`
-	ExpiresIn           int       `json:"expires_in"`
+	Type            string    `json:"token_type"`
+	Scope           string    `json:"scope"`
+	Identity        string    `json:"id_token"`
+	Access          string    `json:"access_token"`
+	ExpiresIn       int       `json:"expires_in"`
+	ExpiresAt       time.Time `json:"expires_at"` // UTC timestamp when access_token expires
+	DefaultTenantID string    `json:"default_tenant_id"`
+}
+
+type TenantToken struct {
```

Comment:
Consider renaming this struct to `TenantSecrets`. It doesn't actually contain a token.
I also think that `pkg/auth0/api` is the best place for this type since it isn't related to auth0 in any way.